docs/book/user-guide/starter-guide/create-an-ml-pipeline.md

```diff
@@ -194,10 +194,10 @@ def svc_trainer(
 ```
 
 {% hint style="info" %}
-If you want to run the step function outside the context of a ZenML pipeline, all you need to do is call the step function outside of a ZenML pipeline. For example:
+If you want to run just a single step on your ZenML stack, all you need to do is call the step function outside of a ZenML pipeline. For example:
```

Comment:
Mhm i'm not sure, mentioning two many alternatives here seems like it would just confuse people in the starter guide, which is why I just opted for the default here. Do you think having explanations for both (and the difference) is necessary here?

Reply:
Hmm i dont get it. the text says "If you want to run just a single step on your ZenML stack, all you need to do is call the step function outside of a ZenML pipeline. For example:" ... but this is wrong right? this will run a pipeline run instead?